src/connection.ts

```diff
@@ -258,7 +258,7 @@ export class ConnectionManager implements IConnectionManager {
           }else{
             attempt = attempt + 1;
             var message = `Timed out connecting to language server. Is the server running at ${address}:${port} ? Will wait timeout value before trying again`;
-            switch(err.code){
+            switch(err['code']){
```

Comment:
This is an object and not a class, there isn't technically a property called `code`. This is essentially the same call, but is accepted by tslint.